consensus/polybft/polybft.go

```diff
@@ -359,6 +361,8 @@ func (p *Polybft) startPbftProcess() {
 
 			return
 		}
+
+		p.logger.Debug("time to run the sequence", "seconds", time.Since(now))
```

Comment:
Are we going to keep this log?

Btw it would be very nice if we could establish open telemetry tracing in IBFT some day. :slightly_smiling_face:

Reply:
It was helpful, I think eventually these logs should be in the engine directly, so it would show directly which stage takes longer.

Reply:
> should be in the engine directly, so it would show directly which stage takes longer.

Yes exactly, that's why I think open telemetry is useful (at least I find it useful back when we were running pbft).

Reply:
Aside from tracing, agree with the log, but also emit a metric?